src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -49,13 +49,43 @@ def set_delete_policy(client, enable=None, days_retained=None):
         policy.days = days_retained
 
     if policy.enabled and not policy.days:
-        from knack.util import CLIError
         raise CLIError("must specify days-retained")
 
     client.set_blob_service_properties(delete_retention_policy=policy)
     return client.get_blob_service_properties().delete_retention_policy
 
 
+def set_service_properties(client, parameters, delete_retention=None, days_retained=None, static_website=None,
+                           index_document=None, error_document_404_path=None):
+    # update
+    kwargs = {}
+    if any([delete_retention, days_retained]):
+        kwargs['delete_retention_policy'] = parameters.delete_retention_policy
+    if delete_retention is not None:
```

Comment:
Would it be clearer to nest the if statements after the `if any()` check? I know it is the same thing, but it might make it more obvious that those if statements are relevant only if the `if any()` is true.

Reply:
I think this may be a code style opinion of mine, but I generally think the less nested if statements the better.